extensions/cornerstone/src/commandsModule.ts

```diff
@@ -618,6 +620,42 @@ function commandsModule({
       );
       toolGroup.setToolEnabled(ReferenceLinesTool.toolName);
     },
+    setDisplaySetToGridViewports: ({ displaySet }) => {
+      const displaySetCache = displaySetService.getDisplaySetCache();
```

Comment:
I would rename this to `setViewportGridDisplaySets` and make it plural since it should be able to set more than one

Reply:
is it for the derivedDisplaySets? then it should be `setDerivedDisplaySetsInGridViewports`